### ISSUE TITLE
Driver performs very slowly when start with log file enabled (appium --log)

### ISSUE THREAD

**Issue by zewer** [appium/appium-windows-driver#279]
Install Appium 2
'npm install -g appium@2.11.4'

Install Windows driver
'appium driver install --source=npm appium-windows-driver@2.12.32'

Open powershell and execute the following command (dont forget to update "user")
`cmd /c start powershell -Command { appium --log C:\Users\<User>\Desktop\appium.log }`

Try to run any desktop test and compare with log file enabled and without.
With log file enabled driver before each command will make 2-4 seconds wait
Was able to reproduce this on Windows 10 local machine and Windows Server pipeline machines

Not reproducible on Appium 1

**Comment by mykola-mokhnach** [comment]
This is weird. Could it be related to some antivirus activity or general i/o slowness?

**Comment by zewer** [comment]
No. I dont have any antivirus on my local VM and without log file everything works as expected

**Comment by zewer** [comment]
Can reproduce on all different machines which I have here (Windows 10/ Windows Server)

**Comment by zewer** [comment]
Appium.WebDriver="4.4.5"
